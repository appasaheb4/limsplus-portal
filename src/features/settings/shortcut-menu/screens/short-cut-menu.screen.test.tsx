import React from 'react';
import {render} from '@testing-library/react';
import {ShortcutMenu} from '.';

describe('ShortcutMenu Screen', () => {
  it('render shortcutMenu correctly', () => {
    const shortcutMenu = render(<ShortcutMenu />);
    expect(shortcutMenu).toMatchSnapshot();
  });
});
