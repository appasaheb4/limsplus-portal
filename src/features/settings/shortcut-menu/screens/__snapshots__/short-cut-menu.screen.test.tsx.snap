// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortcutMenu Screen render shortcutMenu correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sticky top-0 z-20 flex justify-between items-center px-2 py-2 border-b border-gray-300 bg-white"
      >
        <div>
          <h2
            class="text-lg font-bold leading-4 text-gray-900 mt-0"
          />
        </div>
        <div
          class="flex flex-row items-center"
        >
          <img
            alt="banner"
            class="object-fill h-10 w-12 rounded-md mr-2"
          />
          <h2
            class="text-base  leading-4 text-gray-900 mt-0"
          />
        </div>
      </div>
      <hr />
      <div
        class="flex-wrap"
      >
        <label
          class="mt-2"
        >
          List:
        </label>
        <ul
          class="grid grid-cols-6 p-2"
        />
        <br />
        <span
          class="flex flex-row space-x-3   justify-center  items-center"
        >
          <div
            class="undefined"
          >
            <button
              class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
              type="button"
            >
              <svg
                class="-ml-1 -mt-1 mr-2 h-5 w-5 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 7H5a2 2 0 00-2 2v9a2 2 0 002 2h14a2 2 0 002-2V9a2 2 0 00-2-2h-3m-1 4l-3 3m0 0l-3-3m3 3V4"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Update
            </button>
          </div>
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sticky top-0 z-20 flex justify-between items-center px-2 py-2 border-b border-gray-300 bg-white"
    >
      <div>
        <h2
          class="text-lg font-bold leading-4 text-gray-900 mt-0"
        />
      </div>
      <div
        class="flex flex-row items-center"
      >
        <img
          alt="banner"
          class="object-fill h-10 w-12 rounded-md mr-2"
        />
        <h2
          class="text-base  leading-4 text-gray-900 mt-0"
        />
      </div>
    </div>
    <hr />
    <div
      class="flex-wrap"
    >
      <label
        class="mt-2"
      >
        List:
      </label>
      <ul
        class="grid grid-cols-6 p-2"
      />
      <br />
      <span
        class="flex flex-row space-x-3   justify-center  items-center"
      >
        <div
          class="undefined"
        >
          <button
            class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
            type="button"
          >
            <svg
              class="-ml-1 -mt-1 mr-2 h-5 w-5 text-white"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8 7H5a2 2 0 00-2 2v9a2 2 0 002 2h14a2 2 0 002-2V9a2 2 0 00-2-2h-3m-1 4l-3 3m0 0l-3-3m3 3V4"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Update
          </button>
        </div>
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
